### PR TITLE
NF-325 - support for EORI allow list

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -173,3 +173,14 @@ upscan {
 
 timeoutDialog.countdownSeconds = 120
 
+eori {
+    enrolments = [
+      "HMRC-CTS-ORG"
+    ],
+    allowList {
+      enabled = false
```
Comment (crosslandwa) [inline conf/application.conf:181]: CLARIFICATION: The acceptance tests require the allow list to be enabled (they include a `Scenario: user with non-allowed EORI cannot login`)

Does this config conflict with the needs of the acceptance tests?

Comment (SteveSugden) [reply conf/application.conf:181]: Yes.  It does.  Thanks.   I changed this local config after testing the AT.
Shall I enable the allow list in application.conf?  Or remove the AT?

**Review by njmoorewitchcat — APPROVED** [review]

File: conf/application.conf (hunk 1 above)
Comment (crosslandwa) [reply conf/application.conf:181]: I was thinking of commenting against `IdentifierAction.scala` to request extracting/unit testing of the access logic

I think _removing the AT_ and replacing it with unit tests will reduce the overhead of managing this feature (and remove coupling between the AC suite and environment specific config)...

Comment (SteveSugden) [reply conf/application.conf:181]: Done - `AuthenticatedIdentifierActionSpec` enhanced with additional tests

**Review by crosslandwa — APPROVED** [review]

**Pull request merged**